controllers/movies.js

```diff
@@ -0,0 +1,43 @@
+const movies = require('../movies')
+
+const getAllMovies = (request, response) => {
+  return response.send(movies)
+}
+
+const getPartialMatch = (request, response) => {
+  const { input } = request.params
+
+  const matchingMovies = movies.filter(movie => {
+    const stringToPartialMatchTo = (movie.title + ' ' + movie.directors.join(' ')).toLowerCase()
+
+    return stringToPartialMatchTo.includes(input.toLowerCase())
+  })
+
+  return matchingMovies.lenght
```

Comment:
This typo is stopping two of the routes from working. Need to test more closely.

![image](https://user-images.githubusercontent.com/5775716/79990484-1e29f280-847f-11ea-99f5-8d841077a2a2.png)

![image](https://user-images.githubusercontent.com/5775716/79990519-25510080-847f-11ea-8286-1f3f4902b00f.png)